easybuild/easyconfigs/i/IPython/IPython-4.1.0-goolf-1.4.10-Python-2.7.5.eb

```diff
@@ -0,0 +1,161 @@
+    }),
+    ('mistune', '0.7.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/m/mistune/'],
+    }),
+    ('ptyprocess', '0.5', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/ptyprocess/'],
+    }),
+    ('terminado', '0.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/terminado/'],
+    }),
+    ('setuptools_scm', '1.10.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/setuptools_scm/'],
+        'source_tmpl': 'setuptools_scm-1.10.1.tar.bz2',
+    }),
+    ('simplegeneric', '0.8.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/simplegeneric/'],
+        'source_tmpl': 'simplegeneric-0.8.1.zip',
+    }),
+    ('path.py', '8.1.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/path.py/'],
+        'modulename': 'path',
+    }),
+    ('ipython_genutils', '0.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/i/ipython_genutils/'],
+    }),
+    ('pickleshare', '0.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pickleshare/'],
+    }),
+    ('traitlets', '4.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/traitlets/'],
+    }),
+    ('pbr', '1.8.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pbr/'],
+    }),
+    ('funcsigs', '0.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/funcsigs/'],
+    }),
+    ('mock', '1.3.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/m/mock/'],
+    }),
+    ('notebook', '4.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/notebook/'],
+    }),
+    ('jupyter_core', '4.0.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/j/jupyter_core/'],
+    }),
+    ('ipykernel', '4.2.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/i/ipykernel/'],
+    }),
+    ('pyzmq', '15.2.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pyzmq/'],
+        'modulename': 'zmq',
+    }),
+    #('testpath', '0.3', {
+        #'source_urls': ['https://github.com/jupyter/testpath/archive/'], #this extension doesn't provide a setup.py and it's required for the sanity checks (iptest, iptest2)
+        #'source_tmpl': '0.3.tar.gz',
+    #}),
+    ('ipython', version, {
+        'source_urls': ['https://pypi.python.org/packages/source/i/ipython/'],
+        'modulename': 'IPython',
+    }),
+]
+
+modextrapaths = {'PYTHONPATH': ['lib/python%s/site-packages' % pyshortver]}
+
+sanity_check_paths = {
+    'files': ['bin/ipython'],
+    'dirs': ['lib/python%s/site-packages/IPython' % pyshortver],
+}
+
+sanity_check_commands = [
+    ('ipython -h', ''),
+    ('ipython notebook --help', ''),
+    #('iptest', ''), # this test fails with 'ImportError: No module named testpath'. testpath cannot be installed with easyconfig
+    #('iptest2', ''),
```

Comment:
if you run this test you will get an error

```'ImportError: No module named testpath'.```

I cannot install this as an extension because it's distributed as a .whl file

https://pypi.python.org/pypi/testpath/0.3